loader-plugin/src/main/java/com/chiralbehaviors/CoRE/loader/plugin/DatabaseLoader.java

```diff
@@ -30,30 +30,6 @@
 /**
  * @author hhildebrand
  * 
- */
-/*
- * Copyright (c) 2009, 2011 Hal Hildebrand, all rights reserved.
```

Comment:
Ah.  Note that the original had the Chiral Behaviors copywrite.  This was a copy/paste error that I deleted.